wallet/src/de/schildbach/wallet/ui/PaymentsPayFragment.kt

```diff
@@ -195,4 +227,18 @@ class PaymentsPayFragment : Fragment() {
         fun selectContactToPay()
     }
 
+    private fun processResults(data: List<UsernameSearchResult>) {
+        val results = ArrayList<UsernameSearchResult>()
+
+        val contacts = data.filter { /*it.requestSent &&*/ it.requestReceived }
```

Comment:
This is commented out until I determine which contacts should be included.  Currently all payable contacts are included, even those we didn't accept.

Actually, the Live data should do this filter before it posts the results.

Reply:
This has been resolved: https://github.com/dashevo/dash-wallet/pull/475/commits/1cc68c78496e0a2248c64dccb26bd8b16a617016